files/build_templates/sonic_debian_extension.j2

```diff
@@ -724,6 +724,20 @@ sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install {{ name }}=={{
 sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install --from-tarball {{ path }} {{ get_install_options(set_owner, enabled) }}
 {% endfor -%}
 
+{% if install_third_party_debs -%}
+{% for url in install_third_party_debs.strip().split(',') -%}
```

Comment:
@stepanblyschak can we have a macro for this? What's the benefit of having it one string and not the separate commands?

Reply:
Please check new version if that is addressing your comments.